src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,7 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod(Request::METHOD_DELETE)
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ($request->isMethod(Request::METHOD_PUT) &&  '' === $requestContent)
+            || ($request->isMethod(Request::METHOD_PUT) && '' === $requestContent)
```

Comment:
May I suggest:

`|| ($request->isMethod(Request::METHOD_PUT) && '' === ($requestContent = $request->getContent()))`

It will avoid to always retrieve the body (that can be huge) in most cases.